content/en/users/cloud-compute/gpgpu/_index.md

```diff
@@ -33,14 +33,13 @@ dashboards, [VMOps dashboard](../vmops), or [Infrastructure manager](../im).
 It is also possible to use the
 [fedcloudclient](https://fedcloudclient.fedcloud.eu/) for CLI access. Below is
 an example on how to use the `fedcloud` command to show the GPU properties of
-the available flavors:
+the available GPU flavors on all sites:
```

Comment:
this command is for a specific VO, may be worth clarifying that?

Reply:
Could it be "the available GPU flavors on all sites for the specific VO in the command:" ?

Reply:
Addressed in https://github.com/EGI-Federation/documentation/pull/317/commits/301e3eb254c514164b03087118e6c3b0749cfaab

Thanks both!